tests/foreman/cli/test_syncplan.py

```diff
@@ -365,6 +363,33 @@ def test_positive_info_enabled_field_is_displayed(self):
         result = SyncPlan.info({'id': new_sync_plan['id']})
         self.assertIsNotNone(result.get('enabled'))
 
+    @tier1
+    @upgrade
+    def test_positive_info_with_assigned_product(self):
+        """Verify that sync plan info command returns list of products that
+        assigned to that sync plan
```

Comment:
nitpick: missing 'are' before `assigned`? anyway `list of products assigned` or `list of products which are assigned` sounds better imo